src/pages/Game.tsx

```diff
@@ -62,6 +64,17 @@ function Game() {
       socket.emit('leave_room', id);
     };
   }, [id]);
+  useEffect(() => {
+    dispatch(setId(id));
```

Comment:
little confused as to what is going on here. You set the id then do a callback to set id to empty?

Reply:
Yes, we set the ID to the params when the component loads, and when we leave the component the id is empty this helps with changing the ID for the player room